File: articles/algorithms-in-systems-modeling/java/StockFlowUpdate.java
public class StockFlowUpdate {
    public static void main(String[] args) {
        double currentStock = 100.0;
        double inflow = 12.0;
        double outflow = 7.0;
        double nextStock = currentStock + inflow - outflow;
        System.out.printf("next_stock=%.4f%n", nextStock);
    }
}

File: articles/algorithms-in-systems-modeling/fortran/stock_flow_update.f90
program stock_flow_update
  implicit none
  real :: current_stock, inflow, outflow, next_stock
  current_stock = 100.0
  inflow = 12.0
  outflow = 7.0
  next_stock = current_stock + inflow - outflow
  print *, "next_stock=", next_stock
end program stock_flow_update
